extensions/src/AWSSDK.Extensions.NETCore.Setup/ClientFactory.cs

```diff
@@ -219,39 +219,37 @@ private static ClientConfig CreateConfig(Type serviceInterfaceType, AWSOptions o
             }
 
             var defaultConfig = options.DefaultClientConfig;
-            if (options.IsDefaultClientConfigSet)
```

Comment:
This is the one use of the internal `IsDefaultClientConfigSet` which checks to see if the backing field of `DefaultClientConfig` is null. If you read `DefaultClientConfig` like is done above it [sets the value](https://github.com/aws/aws-sdk-net/blob/master/extensions/src/AWSSDK.Extensions.NETCore.Setup/AWSOptions.cs#L79) so this is always true.

I went ahead and deleted the property in order to not have this gotcha sitting around.